packages/cli/test/functional/test_site/expected/index.html

```diff
@@ -314,11 +313,9 @@ <h1 id="heading-with-hidden-keyword"><span id="heading-with-hidden-keyword" clas
           <p><strong>Normal include</strong></p>
           <div>
             <h3 id="establishing-requirements"><span id="establishing-requirements" class="anchor"></span>Establishing Requirements<a class="fa fa-anchor" href="#establishing-requirements" onclick="event.stopPropagation()"></a></h3>
-            <p>
-              <seg id="preview">Requirements gathering, requirements elicitation, requirements analysis,
-                requirements capture are some of the terms commonly <strong>and</strong> interchangeably used to represent the activity
-                of understanding what a software product should do.</seg>
-            </p>
+            <seg id="preview">Requirements gathering, requirements elicitation, requirements analysis,
```

Comment:
>Hmm yep, that would be good. On another note, should we update <seg> to something else in the SSR PR since we don't "allow" unknown components?

sounds good, over here might be better in fact since you've changed the behaviour of unknown tags